JumpscaleCore/servers/threebot/ThreeBotServersFactory.py

```diff
@@ -28,18 +28,19 @@ def _threebot_starting(self, starting=True):
         print("MARK THREEBOT IS STARTING")
 
         j.threebot.active = True
-        if j.core.db and starting:
-            j.core.db.set("threebot.starting", ex=120, value="1")
+        # if j.core.db and starting:
+        #     j.core.db.set("threebot.starting", ex=120, value="1")
         j.data.bcdb._master_set()
         j.servers.myjobs
         j.tools.executor.local
 
-    def threebotserver_check(self):
-        if j.core.db and j.core.db.get("threebot.starting"):
-            self.threebotserver_require()
-            return True
-        res = j.sal.nettools.tcpPortConnectionTest("localhost", 6380, timeout=0.1)
-        return res
+    # FIXME: NOT USED ANYWHERE IS IT REQUIRED ?
+    # def threebotserver_check(self):
+    #     if j.core.db and j.core.db.get("threebot.starting"):
+    #         self.threebotserver_require()
+    #         return True
+    #     res = j.sal.nettools.tcpPortConnectionTest("localhost", 6380, timeout=0.1)
+    #     return res
```

Comment:
@rkhamis also shall we cleanup this too ?

Reply:
yes